src/app/layout.tsx

```diff
@@ -9,14 +9,31 @@ import { getServerSession } from "next-auth";
 import { getLocale } from "./functions/server/l10n";
 import { SessionProvider } from "../contextProviders/session";
 import { authOptions } from "./api/auth/[...nextauth]/route";
+import { getL10n } from "./functions/server/l10n";
 
 const inter = Inter({ subsets: ["latin"] });
 
-export const metadata = {
-  title: "Firefox Monitor",
-  description:
-    "Find out if you’ve been part of a data breach with Firefox Monitor. We’ll help you understand what to do next and continuously monitor for any new breaches.",
-};
+export async function generateMetadata() {
+  const l10n = getL10n();
+  return {
+    title: l10n.getString("brand-fx-monitor"),
+    description: l10n.getString("meta-desc-2"),
+    twitter: {
+      card: "summary_large_image",
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      images: ["/images/og-image.webp"],
+    },
+    openGraph: {
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      siteName: l10n.getString("brand-fx-monitor"),
+      type: "website",
+      url: "https://monitor.firefox.com",
```

Comment:
Q: Does this need to be dynamic so every shared page doesn't link to ye olde [production] homepage? (although I'll ignorantly assume that dev/stage have some "noindex" robots action happening somewheres to avoid those getting SEO'ed)

Reply:
Ah, here's a grep that I think answered at least parts of my ignorant assumptions:

```sh
curl -v --silent https://stage.firefoxmonitor.nonprod.cloudops.mozgcp.net/ 2>&1 | grep "robots"

< x-robots-tag: noindex
```

So it doesn't look like we're using a `<meta name="robots" content="noindex" />` tag, but we are setting `x-robots-tag: noindex` using headers for staging (but not production).

```sh
curl -v --silent https://monitor.firefox.com/ 2>&1 | grep "robots"
# no output
```